api_specifications/evidence-api-1.0.0.yaml

```diff
@@ -0,0 +1,186 @@
+openapi: 3.0.0
+info:
+  title: Evidence
+  description: |
+   API specs for uploading artifacts/marking as evidence etc..Chain of custody and other details of evidence history are not handled in this version. Artifacts can be uploaded voluntarily or because of an order or hearing. They can be marked as "EVIDENCE" by the judge anytime during the course of a case. This is modeled as workflow.
+  version: 1.0.0
+tags:
+  - name: Evidence
+    description: APIs required to upload digital artifacts for evidence consideration
+  
+paths:
+  /artifacts/v1/_create:
+    post:
+      tags:
+        - Evidence
+      summary: Upload artifacts that could potentially be accepted as evidence
+      description: |
+        API used to create a new artifact that could become evidence in a case
+      requestBody:
+        description: Details for the artifact + RequestInfo meta data.
+        content:
+          '*/*':
+            schema:
+              $ref: '#/components/schemas/EvidenceRequest'
+        required: true
+      responses:
+        '201':
+          description: ReponseInfo with case(s) updated sudcessfully
+          content:
+            '*/*':
+              schema:
+                $ref: '#/components/schemas/EvidenceResponse'
+        '400':
+          description: Case(s) creation failed
+          content:
+            '*/*':
+              schema:
+                $ref: >-
+                  https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
+      x-codegen-request-body-name: caseRequest
+  /artifacts/v1/_update:
+    post:
+      tags:
+        - Evidence
+      summary: update case requests.
+      requestBody:
+        description: Details for the new case(s) + RequestInfo meta data.
+        content:
+          '*/*':
+            schema:
+              $ref: >-
+                https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/EvidenceRequest
+        required: true
+      responses:
+        '201':
```

Comment:
typically 201 is only returned when create is done. for all other operations, 200 should be the success code

don't we need a search API as well?

Reply:
Yes, we need search as well. Will add it later.